### ISSUE TITLE
docker build version on development

### ISSUE THREAD

**Issue by bcookatpcsd** [0xERR0R/blocky#787]
preface.. I am not a git aficionado.. I only pretend to use git.. 

```
 docker exec blocky /app/blocky version
[2022-12-05 19:11:43]  WARN 'logTimestamp' is deprecated. Please use 'log.timestamp' instead.
[2022-12-05 19:11:43]  WARN 'port' is deprecated. Please use 'ports.dns' instead.
blocky
Version: v0.18-447-gd501d86
Build time: 20221205-015954
Architecture: amd64
```

The commit tag in the Version is correct.. but the 0.18.. 

I had this happen on a few machines as well.. I needed to rm the clone dir and start over.. 

then I got a build version without the 0.18.. release says 0.20

![image](https://user-images.githubusercontent.com/55087301/205724586-6762ecd9-311c-448d-b9cf-485f441ba97d.png)

right side is git clone --depth 1

left is is git clone

git tag shows up to v0.18 from git clone

git tag shows nothing in --depth 1

Not sure if this is helpful or noise.. 

sorry if it is noise..

**Comment by 0xERR0R** [comment]
Sorry for the confusion. We are using the last tag as version number. I tagged the master branch and built release from this tag. You checked out the development branch. And there is the last tag v0. 18.

**Comment by 0xERR0R** [comment]
related to #744

**Comment by 0xERR0R** [comment]
Fixed via #744. Development branch was deleted. You'll find latest development version on the "main" branch.
